ynput/ayon-core
# Settings: validating transcoding output names

## Changelog Description
Output names needed to be validated since it was causing conflicts downstream during publishing. 

## Testing notes:
1. In studio settings `ayon+settings://core/publish/ExtractOIIOTranscode/profiles`
2. create profile and then output 
3. add name with space between two words and then save the settings. 
4. error message should show you the regex problem.
![image](https://github.com/ynput/ayon-core/assets/40640033/fca603d6-f662-4501-8491-891d4ccace77)
